<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>WidgeTriggerListen</class>
 <widget class="QWidget" name="WidgeTriggerListen">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>300</width>
    <height>99</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QLabel" name="lb_selected">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>24</width>
         <height>24</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>24</width>
         <height>24</height>
        </size>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="pixmap">
        <pixmap resource="../../resources.qrc">:/icons/resources/arrow-right-2-svgrepo-com.svg</pixmap>
       </property>
       <property name="scaledContents">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label">
       <property name="minimumSize">
        <size>
         <width>40</width>
         <height>0</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>40</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="text">
        <string>Listen</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="in_data"/>
     </item>
     <item>
      <widget class="QPushButton" name="bu_delete">
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset resource="../../resources.qrc">
         <normaloff>:/icons/resources/delete-svgrepo-com.svg</normaloff>:/icons/resources/delete-svgrepo-com.svg</iconset>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QRadioButton" name="in_contains">
       <property name="text">
        <string>Contains</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="in_exact">
       <property name="text">
        <string>Exact</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="in_hex">
       <property name="text">
        <string>Hex</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../../resources.qrc"/>
 </resources>
 <connections/>
</ui>
